pallet-chainlink-feed/src/tests.rs

```diff
@@ -603,6 +604,24 @@ fn update_future_rounds_should_work() {
 	});
 }
 
+#[test]
+fn force_admin_transfer_should_work() {
+	new_test_ext().execute_with(|| {
+		let current_admin = ChainlinkFeed::pallet_admin();
+		let new_admin = 42;
+
+		assert_noop!(
+			ChainlinkFeed::force_set_pallet_admin(Origin::signed(current_admin), new_admin),
+			DispatchError::BadOrigin
+		);
+		assert_ok!(ChainlinkFeed::force_set_pallet_admin(
```

Comment:
@mattsse can we also add a failing case, when the method caller is not root?

Reply:
This is already ensured by this check, in which we ensure that even the current admin can't call this.

https://github.com/smartcontractkit/chainlink-polkadot/pull/104/files#diff-34ba86f45aaa350811973171ddae78d326f79cb5b184943e213971dd291f20b0R613-R615